packages/extension-api/src/extension-api.d.ts

```diff
@@ -1657,6 +1659,7 @@ declare module '@podman-desktop/api' {
       callback: (name: string, data: string) => void,
       authInfo?: ContainerAuthInfo,
     ): Promise<void>;
+    export function info(engineId: string): Promise<Info>;
```

Comment:
Here I think we're breaking a pattern as usually we were providing compat API

here it's a Podman API that is returned so I don't know if Info object should return info from the Compat API